StellarDotnetSdk/LedgerEntries/ConfigSettingContractLedgerCostExtV0.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace StellarDotnetSdk.LedgerEntries;
+
+public class ConfigSettingContractLedgerCostExtV0 : LedgerEntryConfigSetting
```

Comment:
Did we normally not have a base class without v1 and have v1 inside of that class and then v2 inside of the v1 class?

This would mean this class would start as 
ConfigSettingContractLedgerCost

Then later we add 
ConfigSettingContractLedgerCosExtV1 as v1 property into ConfigSettingContractLedgerCost

Then later we add 
ConfigSettingContractLedgerCosExtV2 as v2 property into ConfigSettingContractLedgerCostExtV1

etc.

Or do I remember this wrong?

Reply:
Talked on the phone my idea was not possbile kept it as it is right now due to naming conflicts for any logical names due to the generated XDR classed.